### PR TITLE
Update to bevy 0.13

### PR DESCRIPTION
**Pull request by XYCaptain**

### CODE REVIEW

File: src/render/shadow.rs [hunk 1]
```diff
@@ -531,10 +533,12 @@ impl Node for GridShadowPassNode {
                     resolve_target: None,
                     ops: Operations {
                         load: LoadOp::Clear(Color::BLACK.into()),
-                        store: true,
+                        store: StoreOp::Store,//修订
```
Comment (reoseah) [inline src/render/shadow.rs:536]: what is `//修订`?

Comment (XYCaptain) [reply src/render/shadow.rs:536]: Sorry. I noticed it has been updated by [IceSentry](https://github.com/IceSentry)  and I will close this pull request.

Comment (XYCaptain) [reply src/render/shadow.rs:536]: Sorry. I noticed it has been updated by [IceSentry](https://github.com/IceSentry)  and I will close this pull request.

Comment (XYCaptain) [reply src/render/shadow.rs:536]: Sorry. I noticed it has been updated by [IceSentry](https://github.com/IceSentry)  and I will close this pull request.

**Pull request closed without merging**